data/scripts/variants_spdi.py

```diff
@@ -46,26 +47,20 @@ def build_allele(chr, pos, ref, alt, translator):
     return allele
 
 
-def build_spdi_hgvs(chr, pos, ref, alt, translator):
+def build_spdi(chr, pos, ref, alt, translator):
     # Only use translator if the ref or alt is more than one base.
     if len(ref) == 1 and len(alt) == 1:
         chr_ref = CHR_MAP[chr]
         pos_spdi = int(pos) - 1
         # example SPDI: NC_000024.10:10004:C:G
         spdi = f'{chr_ref}:{pos_spdi}:{ref}:{alt}'
-
-        pos_hgvs = f'g.{pos}'
-        # example HGVS: NC_000024.10:g.10005C>G
-        hgvs = f'{chr_ref}:{pos_hgvs}{ref}>{alt}'
     else:
         allele = build_allele(chr, pos, ref, alt, translator)
```

Comment:
It might be possible to also return `valid_ref_seq` from `_from_gnomad`, which should just be `del_seq` you are getting below.